akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -326,7 +326,7 @@ private[akka] class LocalActorRef private[akka] (
    * If this method returns true, it will never return false again, but if it
    * returns false, you cannot be sure if it's alive still (race condition)
    */
-  @deprecated("Use context.watch(actor) and receive Terminated(actor)", "2.2") override def isTerminated: Boolean = actorCell.isTerminated
+  override def isTerminated: Boolean = actorCell.isTerminated
```

Comment:
what, why?
if we can't be without it internally, we can change it to `private[akka]` instead of removing it